packages/contracts/contracts/utils/UncheckedIncrement.sol

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+// Free Function Approach...
+
+// @dev Internal helper methods for unchecked value increments.
+// @param i The uint256 that need to be incremented.
+// @return uin256 incremented without check.
+function _uncheckedIncrement(uint256 i) pure returns (uint256) {
```

Comment:
Why do we need this?

Reply:
yes, it is for saving gas, on increments (for example like loop`s ++i ) that we are sure it will not be big enough for it to overflow